activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Deprecate `(*args)` on `assert_nothing_raised`.
+
+    Using arguments (to specify an exception error) with
```

Comment:
Thank you for mentioning me but we usually don't do that.

What do you think about?

```
Deprecate arguments on `assert_nothing_raised`.

`assert_nothing_raised` were not asserting the arguments that were being passed
(usually an specific exception class) since it only yields the block. To not confuse the
users that the arguments have any meaning they are being deprecated.
```